scripts/main/lychee.js

```diff
@@ -42,6 +42,12 @@ let lychee = {
 	landing_page_enabled: false, // is landing page enabled ?
 	delete_imported: false,
 
+	nsfw_visible: true,
+	nsfw_visible_saved: true,
+	nsfw_blur: false,
+	nsfw_warning: false,
+	nsfw_warning_text: "<b>Sensitive content</b><br><p>This album contains sensitive content which some people may find offensive or disturbing.</p>",
```

Comment:
Shouldn't this be localized?

(as well as in two places further down in this file)

Reply:
Actually, no. It is a config text, so that the user can change it to its liking. :)
Though it is actually unnecessary as this is included in the blade template.